ghost/ghost/src/http/controllers/snippet.dto.input.ts

```diff
@@ -0,0 +1,26 @@
+import {IsString, IsArray, IsNotEmpty, IsOptional, MaxLength, ValidateNested} from 'class-validator';
+import {Type} from 'class-transformer';
+
+class SnippetDTO {
+    @IsString()
+    @IsNotEmpty()
+    @MaxLength(191)
+        name?: string;
+
+    @IsString()
+    @IsOptional()
+        lexical?: string;
```

Comment:
It's annoying that `lexical?: string` literally means it's a string and it's optional, but we have to add decorators for both of these things 😕

Reply:
Yeah... without "IsOptional" it failed the "IsString" validation. Quite verbose this validator is